moonsense/download.py

```diff
@@ -176,30 +244,58 @@ def download(
         session_counter = 0
         filter_by_labels = labels if len(labels) > 0 else None
 
+        max_timestamp_per_day = {}
+
+        print(since, until)
```

Comment:
as an alternative to removing it may be worth using the logging library

Reply:
Changed this to proper printing the dates. We'll need to add the logging library at some point for this too but I think we need to do it everywhere so outside of the scope of this change.